### ISSUE TITLE
Export to RTF

### ISSUE THREAD

**Issue by Dan9er** [Tyrrrz/DiscordChatExporter#24]
TXT files are very compact, but cannot be formatted.

MHTML (#1)/HTML files can look almost exactly like Discord but can be massive in file size.

[RTF files](https://en.wikipedia.org/wiki/Rich_Text_Format) can contain images and basic formatting while taking up less space. It's the best of both worlds!

**Comment by Tyrrrz** [comment]
How about markdown?

**Comment by Dan9er** [comment]
Yeah that's actually more compact than RTF, but I don't think there is a program that can just display MD files in all their rich formatting glory. Still, great idea!

**Comment by Dan9er** [comment]
Word*pad* got you covered there, but there may be others.

**Comment by Tyrrrz** [comment]
Can you insert images in RTF using URL? I wasn't able to do that using Wordpad.

**Comment by Dan9er** [comment]
I don't know... try asking Stack Exchange.

**Comment by 2E0PGS** [comment]
Markdown or LaTeX would be nice. Maybe even CSV? Markdown there is a few libraries for converting to and from it.

**Comment by 2E0PGS** [comment]
Try looking into https://github.com/jgm/pandoc for Markdown.

**Comment by Tyrrrz** [comment]
Gonna narrow the scope to just RTF since it seems like the best option.
